tox.ini

```
@@ -114,6 +114,7 @@ envlist =
     python-external_http-{py37,py38,py39,py310,py311,py312,py313},
     python-external_httplib-{py37,py38,py39,py310,py311,py312,py313,pypy310},
     python-external_httplib2-{py37,py38,py39,py310,py311,py312,py313,pypy310},
+    python-external_pyzeebe-{py37,py38,py39,py310,py311,py312,py313,pypy310},
```

```suggestion
    python-external_pyzeebe-{py39,py310,py311,py312,pypy310},
```
Updating this based on the pyzeebe supported Python versions listed on PyPI:  https://pypi.org/project/pyzeebe/